ts/components/staking/wizard/wizard_flow.tsx

```diff
@@ -710,7 +715,7 @@ export const TokenApprovalPane = (props: TokenApprovalPaneProps) => {
             <ButtonWithIcon
                 onClick={async () => {
                     const allowanceBaseUnits =
-                        (providerState.account as AccountReady).zrxAllowanceBaseUnitAmount || new BigNumber(0);
+                        ({ address: account } as AccountReady).zrxAllowanceBaseUnitAmount || new BigNumber(0);
```

Comment:
`zrxAllowanceBaseUnitAmount` won't exist on `address`, need to port over from the redux store still